src/fheroes2/heroes/heroes.cpp

```diff
@@ -1818,12 +1824,17 @@ const fheroes2::Sprite & Heroes::GetPortrait( int id, int type )
             return mediumSizePortrait.try_emplace( id, std::move( output ) ).first->second;
         }
         case PORT_SMALL:
-            return Heroes::DEBUG_HERO > id ? fheroes2::AGG::GetICN( ICN::MINIPORT, id ) : fheroes2::AGG::GetICN( ICN::MINIPORT, BRAX );
+            if ( id == Heroes::DEBUG_HERO ) {
+                return fheroes2::AGG::GetICN( ICN::MINIPORT, BRAX );
```

Comment:
ICN index for the small portrait of Debug hero should also be deducted by 1.
```suggestion
                return fheroes2::AGG::GetICN( ICN::MINIPORT, BRAX - 1 );
```

Reply:
Hi @Districh-ru , I don't understand. Why we need to change the ICN value as this pull request does not touch images?

Reply:
Ah, my bad. You are absolutely right!